docs/querying/api.md

```diff
@@ -902,6 +906,10 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 {
   "status": "success",
   "data": {
+    "numSeries": 508,
+    "chunkCount": 937,
```

Comment:
I'm not sure what to do about these two fields, as they are also reported by `/api/v1/status/runtimeinfo` already (see https://prometheus.demo.do.prometheus.io/api/v1/status/runtimeinfo), and exposing the same stats in two different endpoints seems like suboptimal API design.

Probably they shouldn't be part of `runtimeinfo` nowadays, but just part of the new `tsdb` stats.

How about removing them from the existing `runtimeinfo` stats, as the docs for that endpoint above don't even mention those fields (among other missing ones), and there's a notice saying that fields are allowed to change between Prometheus versions.